### DISCUSSION TITLE
Can't change the app locale. What am I doing wrong?

### DISCUSSION THREAD

**Discussion by CorvS** [statamic/cms#11291] [Q&A] [answered]
Hello everyone 👋

It's probably another one of those super obvious things and I just overlooked it. I want to create a website that is only in German, but when I change my app locale to `de`, it's still `en`.

- Fresh Statamic install (using `statamic new`)
- Tried changing the env variable: `APP_LOCALE=de`
- Tried changing the app config directly: `'locale' => 'de'`
- My config is not cached and I tried clearing it multiple times
- When using Tinker and `config('app.locale')` it shows the correct locale `de`
- But, when I use `{{ config:app:locale }}` in any of my Antlers templates it shows `en` and my validation messages are also in English

Please tell me what I did wrong. 😄

Cheers

**Comment by jasonvarga** [comment, marked as answer]
You will need to edit the `locale` of your "site".

This is located in `resources/sites.yaml` or you can do it via the Site section in the control panel.

Make sure _that_ is set to `de_DE`.

What Statamic does is when you visit the front-end, it will update the app locale for that request to the one you've configured for that site.

**Reply by CorvS** [reply]
Thank you!
